pkg/oci/ccm.go

```diff
@@ -42,6 +43,13 @@ import (
 	"github.com/oracle/oci-cloud-controller-manager/pkg/oci/util"
 )
 
+const (
+	// RateLimitQPSDefault sets a value for the default queries per second.
+	RateLimitQPSDefault = 20.0
+	// RateLimitBucketDefault sets a value for the default token bucket burst size.
+	RateLimitBucketDefault = 5
```

Comment:
Doesn't need to start with a capital (which reduces req for a comment too) as i don't think it's used outside this pkg?